lib/handlers/health-check.ts

```diff
@@ -38,6 +38,7 @@ export class HealthCheckHandler {
         private readonly compilationQueue: CompilationQueue,
         private readonly filePath: any,
         private readonly compileHandler: ICompileHandler,
+        private readonly isExecutionWorker: boolean,
```

Comment:
Maybe instead of "isBlah" lets have an `enum` of the node type... I can imagine once we have this working we might reasonably start having different node types (maybe "frontend", "compiler", "executor"?)

Mostly this is my railing against another boolean thing in a list of params for readability though: `new Blah(true)` vs `new Blah(NodeType.execution)`

Reply:
I understand the question, but the consequences design-wise are complicated. We can change it when we add a new type.

Reply:
Ok. I'm just trying not to have too many `bool`s around in general.